1.wave2_data/3.processes_features/extract_image_features_interstellar.py

```diff
@@ -83,10 +87,60 @@
 image_features_df.head()
 
 
-# ## View info of the dataframe
+# ## Merge platemap metadata with extracted image features
 
 # In[8]:
 
 
+image_features_df = annotate(
+    profiles=image_features_df,
+    platemap=platemap_df,
+    join_on=["Metadata_well", "Image_Metadata_Well"],
+    output_file="none",
+)
+
+
+# ## Add condition as a metadata column to the dataframe
+
+# In[9]:
+
+
+def conditions(well_column):
```

Comment:
Consider providing a more descriptive name for this function to help describe the purpose.

Reply:
Done!